README.md

```diff
@@ -1,2 +1,49 @@
-# Main-2022-09-22
- 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse Project
+
+> For this project, I had to create "Hello world" repository and a feature branch that i cloned to my local machine, I then added index.HTML file and a Styles.css File, I set up linters for HTML&CSS and then created a .gitignore file to make sure certain files do not get published when i commit my changes from my local repo to my GitHub Master Repo, The goal here is to master all of the tools and best practices learned about in previous steps of The Program, which I will be using in all Microverse projects and most likely in my future job as well, so it is important to know the processes required!
+
+
+## Built With
+
+- HTML & CSS
+- Github Workflow Actions, HTML&CSS Linters
+- Virtual Studio Code, GitHub
+
+
+
+## Getting Started
+
+**This Project was to make sure i understand the basic tools and methods i will be using on a daily basis during the course of my microverse programm as well as future work life.**
+
+
```

Comment:
[OPTIONAL] Here, you are suppose to give information on how to clone the project, how to start or open the app, and what technologies or devices are needed. It should be something similar to the image below.

![Screenshot from 2022-09-23 08-52-35](https://user-images.githubusercontent.com/65233860/191925238-3c6e01a5-14e6-4298-937f-9074797e93c1.png)

Reply:
Thank you for the further insight into these issues, I really do appreciate it, I have resolved them